.travis.yml

```diff
@@ -18,30 +19,37 @@ gemfile:
   - gemfiles/rails5.1.gemfile
   - gemfiles/rails5.2.gemfile
 matrix:
-  include:
-    - rvm: jruby-9.1.1.0
-    - rvm: jruby-head
-    - rvm: ruby-head
   exclude:
     - rvm: 2.1.10
       gemfile:
         - gemfiles/rails5.0.gemfile
+    - rvm: 2.1.10
```

Comment:
Why do you need to repeat this when `gemfile` key accepts an array. This seems like a huge duplicate effort?

Reply:
In my experience it doesn't take an array unfortunately. If you look back at the earlier travis test runs it was ignoring the exclusions when the gemfiles were provided as an array. I wish it didn't because that would be simpler. We found this with `acts_as_list` too.